src/main/java/com/sse/ooseproject/models/Person.java

```diff
@@ -1,23 +1,56 @@
 package com.sse.ooseproject.models;
 
-import jakarta.persistence.*;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.MappedSuperclass;
+import jakarta.persistence.GenerationType;
 
-public class Person {
-    // Entities or MappedSuperclass models in Spring require an id. The @GeneratedValue annotation makes sure
-    // that the id is automatically increased when inserting new objects into the database.
+@MappedSuperclass
+public abstract class Person {
+    //Properties
     @jakarta.persistence.Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     private long id;
-
-    // TODO add more attributes.
+    private String firstName;
+    private String lastName;
+    private String emailAddress;
```

Comment:
Hier fehlt 
`@Column(name="email")`

Gerade in der db geschaut, da heißt das Feld email statt email_Address

Reply:
oder natürlich die property einmal renamen übermal